terraform.tfvars.json.sample

```diff
@@ -1,25 +1,24 @@
 {
-    "name_prefix": "<YOUR UNIQUE PREFIX NAME TO BE CREATED>",
-    "dns_provider": "azure",
-    "tsb_fqdn": "<YOUR UNIQUE PREFIX NAME TO BE CREATED>.azure.sandbox.tetrate.io",
-    "tsb_version": "1.6.0",
-    "tsb_image_sync_username": "<TSB_REPO_USERNAME>",
-    "tsb_image_sync_apikey": "<TSB_REPO_APIKEY>",
-    "tsb_password": "Tetrate123",
-    "tsb_mp": {
-        "cloud": "azure",
-        "cluster_id": 0
-    },
-    "tsb_org": "tetrate",
-    "aws_k8s_regions": [
-    ],
+    "aws_k8s_regions": [],
     "azure_k8s_regions": [
         "eastus"
     ],
+    "dns_provider": "azure",
```

Comment:
this is an optional property, let's not promote it to required.

Reply:
Check :heavy_check_mark: